pointsource_imaging/NIRCam_Basic_PSF_Photometry_solution.ipynb

```diff
@@ -74,7 +74,7 @@
    "id": "95891849",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    #os.environ['WEBBPSF_PATH'] = '/data/webbpsf-data'
</pre><p>&nbsp;This line should either be deleted or check whether the <code>$WEBBPSF_PATH</code> environment variable already exists.</p><p> </p><p>If you choose the latter, you could try something like:</p><pre class="ql-syntax" spellcheck="false">if not os.environ.get('WEBBPSF_PATH'):
    os.environ['WEBBPSF_PATH'] = '/data/webbpsf-data' 
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>